types/node/index.d.ts

```diff
@@ -5381,8 +5381,8 @@ declare module "stream" {
             highWaterMark?: number;
             decodeStrings?: boolean;
             objectMode?: boolean;
-            write?: (chunk: string | Buffer, encoding: string, callback: Function) => any;
-            writev?: (chunks: Array<{ chunk: string | Buffer, encoding: string }>, callback: Function) => any;
+            write?: (chunk: any, encoding: string, callback: Function) => any;
```

Comment:
is it really `any` here or should this be something like `{}` to avoid `null` and `undefined`?

Reply:
I have read here https://nodejs.org/dist/latest/docs/api/stream.html#stream_writable_write_chunk_encoding_callback that `null` is not allowed in object mode. But you are right the APIs should be symmetric.